boxoffice/templates/ticket_assignment_mail.html

```diff
@@ -105,25 +127,64 @@
 
     <p>Hello {{ line_item.current_assignee.fullname }},</p>
 
-    <div class="wrapper">
-      <table class="ticket-table"> 
-        <tr>
-          <td class="ticket-title-wrapper">
-            <p class="text-center ticket-title">{{ line_item.item.title }}</p>
-            <p class="receipt-no">Receipt No: {{order.invoice_no}}</p>
-          </td>
-
-          <td class="details-wrapper">
-            <p class="text-center fullname">{{ line_item.current_assignee.fullname }}</p>
-            {%- if line_item.current_assignee.details.company -%}<p class="text-center company">{{ line_item.current_assignee.details.company }}</p>{%- endif %}
-            {%- if line_item.current_assignee.details.twitter -%}<p class="text-center twitter-handle">@{{ line_item.current_assignee.details.twitter | replace("@", "") }}</p>{%- endif %}
-          </td>
-        </tr>
-      </table>
+    <div class="ticket-wrapper ">
+      <div class="ticket">
+        <div class="header">
+          {%- if item_collection.details.logo -%}
+            <img src="{{ item_collection.details.logo }}" class="logo"/>
+          {%- elif org.details.logo %}
+            <img src="{{ org.details.logo }}" class="logo"/>
+          {%- endif %}
+          <div class="event">
+            <p class="event-title">{{ item_collection.title }}</p>
+            <p class="event-date">
+            {%- if line_item.item.details.date -%}
+              <span>{{ line_item.item.details.date }}</span>
```

Comment:
This is a slightly inefficient way to access data via a relationship. Since this doesn't impact user experience it's probably fine, but ideally you might want to fetch all the data you need with a join in the view, and pass the data to the template.